### PR TITLE
telegraf: Release v1.31.0

### LINKED ISSUE #5455: Release notes page missing 1.30.2
The release notes page does not describe Telegraf 1.30.2 released on 2024-04-22.

##### Relevant URLs
- https://docs.influxdata.com/telegraf/v1/release-notes/
- Release page for Telegraph 1.30.2: https://github.com/influxdata/telegraf/releases/tag/v1.30.2

### PR DESCRIPTION
**Pull request by powersj**
Adds all missing change log entries as well.

Closes #5455

_Describe your proposed changes here._

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [x] Rebased/mergeable

### CODE REVIEW

File: content/telegraf/v1/plugins.md [hunk 1]
```diff
@@ -9,61 +9,11 @@ menu:
...
-  - /telegraf/v1/plugins/aggregators/
-  - /telegraf/v1/plugins/inputs/
-  - /telegraf/v1/plugins/outputs/
-  - /telegraf/v1/plugins/processors/
+  - /telegraf/v1/plugins/plugin-list/
   - /telegraf/v1/plugins/plugins-list/
-  - /telegraf/v1/plugins/aggregators/
-  - /telegraf/v1/plugins/inputs/
-  - /telegraf/v1/plugins/outputs/
   - /telegraf/v1/plugins/processors/
```
Comment (sanderson) [inline content/telegraf/v1/plugins.md:17]: These must be relics from when we documented by minor version. A little find/replace action that never got fully cleaned up.

File: content/telegraf/v1/release-notes.md [hunk 2]
```diff
@@ -571,6 +1229,113 @@ can help with migrating to newer plugins.
...
+- Update `github.com/aws/aws-sdk-go-v2/service/sts` from 1.18.5 to 1.18.9.
+- Update `github.com/docker/docker` from 23.0.0 to 23.0.4.
+- Update `github.com/openconfig/gnmi` from 0.0.0-20220920173703-480bf53a74d2 to 0.9.1.
+- Update `github.com/prometheus/common` from 0.41.0 to 0.42.0.
+- Update `github.com/safchain/ethtool` from 0.2.0 to 0.3.0.
+- Update `github.com/tinylib/msgp` from 1.1.6 to 1.1.8.
+- Update `github.com/vishvananda/netns` from 0.0.2 to 0.0.4.
+- Update `github.com/wavefronthq/wavefront-sdk-go` from 0.11.0 to 0.12.0.
+
+## v1.26.1 {date="2023-04-3"}
```
Comment (sanderson) [inline content/telegraf/v1/release-notes.md:1296]:
```suggestion
## v1.26.1 {date="2023-04-03"}
```

**Review by sanderson — APPROVED** [review]
Thanks for this @powersj. Not sure how we were missing so much, but thanks for filling in the gaps. Just one minor suggestions and it's gtg! 🚀

File: content/telegraf/v1/plugins.md (hunk 1 above)
Comment (powersj) [reply content/telegraf/v1/plugins.md:17]: Agreed - I figured it was cool to clean this long list up to just the unique values.

Comment (powersj) [reply content/telegraf/v1/plugins.md:17]: Agreed - I figured it was cool to clean this long list up to just the unique values.

**Pull request merged**